Nodejs/Product/Nodejs/Nodejs.csproj

```diff
@@ -1684,27 +1685,18 @@
   </Target>
   <PropertyGroup>
     <PostBuildEvent>
-      copy /y $(OutDir)..\WebRole\Microsoft.NodejsTools.WebRole.dll $(OutDir)Debugger\RemoteDebug\
+      copy /y $(OutDir)..\WebRole\Microsoft.NodejsTools.WebRole.dll $(OutDir)RemoteDebug\
```

Comment:
Yes, this change doesn't solve the missing webrole.dll problem yet. I'm still trying to figure out the best way to address it but may take that in with another change.